tests/Microsoft.Identity.Test.Performance/TokenCacheTests.cs

```diff
@@ -45,10 +36,8 @@ public class TokenCacheTests
         // This is a workaround to specify the exact param combinations to be used.
         public IEnumerable<(int, int)> CacheSizeSource => new[] {
             (1, 10),
-            (1, 10000),
```

Comment:
What is the reason to remove some combinations? How are these chosen?

Reply:
The goal is to have some common, unique scenarios. The perf is essentially the same for cases with any number of tenants. And then also two cases that test with small and larger number of tokens per tenant.